### PR TITLE
adds logging to ripley gibs

### PR DESCRIPTION
**Pull request by ghost**
👏 BACK 👏 TO 👏 HIPPIE 👏 

okay but for real lets log gibbings okay?

closes #32025

### CODE REVIEW

**Comment by Arianya** [comment]
You currently have a negative Fix/Feature pull request delta of -1. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Arianya** [comment]
:3c

**Comment by Armhulen** [comment]
shoo shoo it's logging, which is like 1 token right

File: code/game/mecha/equipment/tools/mining_tools.dm [hunk 1]
```diff
@@ -82,6 +82,7 @@
 	add_logs(user, target, "attacked", "[name]", "(INTENT: [uppertext(user.a_intent)]) (DAMTYPE: [uppertext(damtype)])")
 	if(target.stat == DEAD)
 		if(target.butcher_results)
+			add_logs(user, target, "gibbed", "[name]")
```
Comment (Cyberboss) [inline code/game/mecha/equipment/tools/mining_tools.dm]: add_logs(user, target, "gibbed", name)

**Review by Cyberboss — DISMISSED** [review]

File: code/game/mecha/equipment/tools/mining_tools.dm (hunk 1 above)
Comment (Armhulen) [reply code/game/mecha/equipment/tools/mining_tools.dm]: thank you

**Comment by Repukan** [comment]
You currently have a negative Fix/Feature pull request delta of -1. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by praisenarsie** [comment]
its not funny when its a pr that gives points..............

**Comment by Armhulen** [comment]
^^^^^

**Pull request merged**